### ISSUE TITLE
After importing a Scatter wallet, I can't unlock the wallet

### ISSUE THREAD

**Issue by maxocoin** [greymass/anchor#929]
I successfully imported a Scatter 11 wallet with Anchor 1.0.5
During the import process I was asked only for one password, the password for decipher the Scatter backup.

After the import process the Anchor wallet shows locked.
When I try to unlock it, I type the Scatter backup password (the only used in the whole process) and it says it is the wrong password

**Comment by aaroncox** [comment]
We're still fighting with this issue. There's some sort of backup configuration coming out of Scatter we haven't been able to reproduce yet.

**Comment by maxocoin** [comment]
A way to get around that could be to add a "import scatter to the current wallet" option.

This could add the all scatter private keys from the backup to the current Anchor Wallet (like the current Tools/Manage Wallets/Import a Private Key)  and we already have a password for the current account  :-)

**Comment by aaroncox** [comment]
True, that might actually be a good way to do it. Sort of an "import and merge" feature. 

We'll keep plunking away at improving this feature, but I think by the time we actually get it working most users might have actually just imported manually 😂
